xml/admin_ceph_gateway.xml

```diff
@@ -3279,365 +3211,211 @@ rgw_zone=us-east-1</screen>
+         <para>Restart the &cogw; in the secondary zone:</para>
+<screen>&prompt.cephuser.ogw;systemctl restart ceph-radosgw@rgw.`hostname -s`</screen>
+       </step>
+     </procedure>
+   </sect2>
+   <sect2 xml:id="ceph-rgw-single-site-multi">
+     <title>Migrating a Single Site System to Multi-Site</title>
+     <para>To migrate from a single site system with a default &zonegroup;
+       and zone to a multi site system, use the following steps:</para>
+     <procedure>
+       <step>
+         <para>Create a realm. Replace <replaceable>NAME</replaceable> with
+         the realm name:</para>
+<screen>&prompt.cephuser;radosgw-admin realm create --rgw-realm=<replaceable>NAME</replaceable> --default</screen>
+       </step>
+       <step>
+         <para>Rename the default zone and &zonegroup;. Replace
+           <replaceable>NAME</replaceable> with the &zonegroup; or zone name.</para>
+<screen>&prompt.cephuser;radosgw-admin zonegroup rename --rgw-zonegroup default --zonegroup-new-name=<replaceable>NAME</replaceable>
+&prompt.cephuser;radosgw-admin zone rename --rgw-zone default --zone-new-name us-east-1 --rgw-zonegroup=<replaceable>NAME</replaceable></screen>
+       </step>
+       <step>
+         <para>Configure the master &zonegroup;. Replace <replaceable>NAME</replaceable> with the realm
+           or &zonegroup; name. Replace <replaceable>FQDN</replaceable> with the fully
+           qualified domain name(s) in the &zonegroup;.</para>
+<screen>&prompt.cephuser;radosgw-admin zonegroup modify --rgw-realm=<replaceable>NAME</replaceable> --rgw-zonegroup=<replaceable>NAME</replaceable> --endpoints http://<replaceable>FQDN</replaceable>:80 --master --default</screen>
+       </step>
+       <step>
+         <para>Configure the master zone. Replace <replaceable>NAME</replaceable> with the realm, &zonegroup;
+           or zone name. Replace <replaceable>FQDN</replaceable> with the fully
+           qualified domain name(s) in the &zonegroup;.</para>
+<screen>&prompt.cephuser;radosgw-admin zone modify --rgw-realm=<replaceable>NAME</replaceable> --rgw-zonegroup=<replaceable>NAME</replaceable> \
+                            --rgw-zone=<replaceable>NAME</replaceable> --endpoints http://<replaceable>FQDN</replaceable>:80 \
+                            --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET-KEY</replaceable> \
+                            --master --default</screen>
+       </step>
+       <step>
+         <para>Create a system user. Replace <replaceable>USER-ID</replaceable> with the username.
+           Replace <replaceable>DISPLAY-NAME</replaceable> with a display name.
+           It may contain spaces.</para>
+<screen>&prompt.cephuser;radosgw-admin user create --uid=<replaceable>USER-ID</replaceable> --display-name="<replaceable>DISPLAY-NAME</replaceable>"\
+                            --access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET-KEY</replaceable> --system</screen>
```

Comment:
Should be create the sync user before modifying the master zone above, so we have the access and secret keys?

Reply:
agree, move this before zone creation so that `--access-key`  `--secret` can be used in zone create itself